### PR TITLE
Resize minimap when browser resize occurs

### PR DESCRIPTION
**Pull request by dbunker**

### CODE REVIEW

**Comment by dbunker** [comment]
Example of issue can be seen [here](http://www.wavesurfer.fm/example/annotation/). First, reload the tab when the browser is small, then expand the browser to full size. You can see the minimap remains at the original size. This pull checks when a resize occurred based on the window resize event and the container size, then re-renders if needed.

File: plugin/wavesurfer.minimap.js [hunk 1]
```diff
@@ -117,6 +117,14 @@ WaveSurfer.Minimap = WaveSurfer.util.extend({}, WaveSurfer.Drawer, WaveSurfer.Dr
         }
 
         this.wavesurfer.on('destroy', this.destroy.bind(this));
+
+        my.check_width = 0;
```
Comment (katspaugh) [inline plugin/wavesurfer.minimap.js]: Please change this to `var prevWidth = 0;`.

File: plugin/wavesurfer.minimap.js [hunk 2]
```diff
@@ -117,6 +117,14 @@ WaveSurfer.Minimap = WaveSurfer.util.extend({}, WaveSurfer.Drawer, WaveSurfer.Dr
...
+        window.addEventListener('resize', function() {
```
Comment (katspaugh) [inline plugin/wavesurfer.minimap.js]: Please add a space after `function`.

Comment (katspaugh) [inline plugin/wavesurfer.minimap.js]: You need to also unsubscribe from the resize event on destroy.

```
this.on('destroy', function () {
    window.removeEventListener(onResize);
});
```

**Comment by katspaugh** [comment]
Mörged! :metal:

**Pull request merged**